prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+    /* stylelint-disable block-no-empty */
+    body {
+      font-family: sans-serif;
+    }
+
+    h1 {
+      text-align: center;
+    }
+
+    .container {
+      max-width: 992px;
+      margin: auto;
+    }
+
+    .row {
+      width: 100%;
+      display: flex;
+    }
+
+    .column {
+      flex-basis: calc(100% / 3);
+      padding-left: 8px;
+      padding-right: 8px;
+    }
+
+    img {
+      width: 100%;
+      height: 450px;
+      object-fit: contain;
+    }
+  </style>
+</head>
+<body>
+<h1>Indiana Jones</h1>
+  <div class="container"></div>
+    <div class="row"></div>
+      <div class="column">
+        <img src="images/raiders.jpg" alt="Raiders of the Lost Ark">
+        <h3>Raiders of the Lost Ark</h3>
+        <p>Lucas conceived Raiders of the Lost Ark in the arly 1970s. Seeking to modernize the serial films of the early 20th-century, he developed the idea further with Kaufman, who suggested the Ark as the film's goal. Lucas eventually focused on developing his 1977 space opera Star Wars.</p>
+</div>
+
+<div class="column">
+  <img src="images/temple.jpg" alt="Temple of Doom">
+  <h3>Temple of Doom</h3>
+    <p>The film was released to financial success but initial reviews were mixed, criticizing its darker elements and violence. However, critical opinion has improvedsince 1984, citing the film's intensity and imagination.</p>
+</div>
+
+<div class="column">
+  <img src="images/crusade.jpg" alt="The Last Crusade">
+  <h3>The Last Crusade</h3>
+    <p>After the mixed reaction to Indiana Jones and the Temnple of Doom, Spielberg chose tot one down the dark tone and grpahic violence in the next installment. During the five years between The Temple of doom and The Last Crusade, he and executive producer Lucas reviewed several scripts before accepting Jeffrey Boam's.</p>
+</div>
+
+
```

Comment:
You are indeed missing closing `</div>` tags. Every opening `<div>` tag has to have a matching closing tag. Proper indenting helps a lot with making sure all your tags are properly closed.